core/src/main/java/de/danoeh/antennapod/core/util/playback/Playable.java

```diff
@@ -223,6 +228,8 @@ public static Playable createInstanceFromPreferences(Context context, int type,
                 case ExternalMedia.PLAYABLE_TYPE_EXTERNAL_MEDIA:
                     result = createExternalMediaInstance(pref);
                     break;
+                default:
+                    result = null;
```

Comment:
Please also add a break statement

Reply:
After starting from scratch I didn't touch that section any more.